client/lib/plans/constants.js

```diff
@@ -1119,19 +1119,19 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_FREE_BLOG_DOMAIN ]: {
 		getSlug: () => FEATURE_ADVANCED_CUSTOMIZATION,
-		getTitle: () => i18n.translate( 'Free .blog domain' ),
+		getTitle: () => i18n.translate( 'Free .blog Domain for One Year' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Free .blog domain for one year' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).